### PR TITLE
add Setters to Compound Types Properties

### PR DESCRIPTION
**Pull request by rochar**
EF needs to set data retrieved from database
create test to ensure data is readed from the db correctly

### CODE REVIEW

**Review by alexandervlasenko — DISMISSED** [review]

File: src/Nox.Types/Types/LatLong/LatLong.cs [hunk 1]
```diff
@@ -10,9 +10,16 @@ public sealed class LatLong : ValueObject<(double Latitude, double Longitude), L
...
 
-    public double Latitude => Value.Latitude;
-
-    public double Longitude => Value.Longitude;
+    public double Latitude
+    {
+        get => Value.Latitude;
+        private set => Value = (Latitude: value, Longitude: Value.Longitude);
+    }
+    public double Longitude
```
Comment (ILoveMyTortoise) [inline src/Nox.Types/Types/LatLong/LatLong.cs:19]: nit: space can be added in between

File: tests/Nox.Types.Tests/EntityFrameworkTests/NoxTypesEntityFrameworkTests.cs [hunk 2]
```diff
@@ -1,7 +1,12 @@
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+
 namespace Nox.Types.Tests.EntityFrameworkTests;
 
 public class NoxTypesEntityFrameworkTests : TestWithSqlite
 {
+ 
+
```
Comment (ILoveMyTortoise) [inline tests/Nox.Types.Tests/EntityFrameworkTests/NoxTypesEntityFrameworkTests.cs:9]: why space?

File: tests/Nox.Types.Tests/EntityFrameworkTests/TestWithSqlite.cs [hunk 3]
```diff
@@ -6,23 +6,39 @@ namespace Nox.Types.Tests.EntityFrameworkTests;
 public abstract class TestWithSqlite : IDisposable
 {
     private const string InMemoryConnectionString = "DataSource=:memory:";
+    //private const string InMemoryConnectionString = @"DataSource=test.db";
```
Comment (ILoveMyTortoise) [inline tests/Nox.Types.Tests/EntityFrameworkTests/TestWithSqlite.cs:9]: why add this?

File: tests/Nox.Types.Tests/EntityFrameworkTests/TestWithSqlite.cs [hunk 4]
```diff
@@ -6,23 +6,39 @@ namespace Nox.Types.Tests.EntityFrameworkTests;
...
     private readonly SqliteConnection _connection;
 
-    protected readonly SampleDbContext DbContext;
+    protected SampleDbContext DbContext;
 
     protected TestWithSqlite()
     {
+        var c = Environment.CurrentDirectory;
```
Comment (ILoveMyTortoise) [inline tests/Nox.Types.Tests/EntityFrameworkTests/TestWithSqlite.cs]: why this c?

**Review by ILoveMyTortoise — DISMISSED** [review]

**Review by jan-schutte — DISMISSED** [review]

File: tests/Nox.Types.Tests/EntityFrameworkTests/TestWithSqlite.cs (hunk 3 above)
Comment (rochar) [reply tests/Nox.Types.Tests/EntityFrameworkTests/TestWithSqlite.cs:9]: just for debugging proposes, I will remove

File: tests/Nox.Types.Tests/EntityFrameworkTests/TestWithSqlite.cs (hunk 4 above)
Comment (rochar) [reply tests/Nox.Types.Tests/EntityFrameworkTests/TestWithSqlite.cs]: will remove, just to see where the database file was being saved

**Pull request merged**